packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/ChangesIndicator.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex } from "theme-ui";
+
+interface ChangesIndicatorProps {
+  numberOfChanges: number;
+}
+
+export const ChangesIndicator = ({
+  numberOfChanges,
+}: ChangesIndicatorProps) => {
+  if (numberOfChanges === 0) {
+    return null;
+  }
+  return (
+    <Flex
+      sx={{
+        borderRadius: "50%",
+        backgroundColor: "#6548FF1A",
+        width: "24px",
+        height: "24px",
+        justifyContent: "center",
+        alignItems: "center",
+        fontSize: "12px",
+        color: "#5D40F7E5",
```

Comment:
should we use theme ui or something for the colors ?

Reply:
Is there an example of where this is used already?

Reply:
`slice-machine/packages/slice-machine/components/Forms/components/FlexCard.tsx`
Seems easier than i thought 😮